org-configurations/default-configuration.yml

```diff
@@ -4,6 +4,7 @@ slug: "default"
 
 settings:
   mobile_phone_verification: false
+  identity_verification: false
```

Comment:
if we can simplify as suggested in the previous comment, this will not be necessary